server/tnt_workdir/api/townapi.lua

```diff
@@ -12,6 +12,17 @@ local COL_TOWN_CP_COUNT = 9
 
 local MAX_TOWNS_BATCH_SIZE = 1024
 
+local TOWN_HAS_METRO = {4}
```

Comment:
lua table can be used as map and will simplify logic:
`local TOWN_HAS_METRO = {
    [4] = true
}`

`if TOWN_HAS_METRO[townid] then
... your staff there ...
end`